composer.json

```diff
@@ -1,6 +1,11 @@
 {
-  "require": {
-    "php": ">=5.3.0",
-    "dompdf/dompdf": "*"
-  }
+	"repositories": [
+		{
+		"type": "git",
+		"url": "https://github.com/dompdf/dompdf.git"
+		}
+	],
+	"require": {
+		"dompdf/dompdf": "dev-master#45d260f2f6c6a542e0db3459470374c986a46d93"
```

Comment:
why use dev-master instead of the official 1.0.1 release? (or the soon-to-be-released 1.0.2)